pos_mail_receipt/models/pos_order.py

```diff
@@ -0,0 +1,70 @@
+# Copyright 2019 Coop IT Easy SCRLfs
+# @author Pierrick Brun <pierrick.brun@akretion.com>
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+
+
+import base64
+from odoo import fields, models, api, _
+
+
+class PosOrder(models.Model):
+    _inherit = "pos.order"
+
+    email_receipt_sent = fields.Boolean()
+
+    @api.model
+    def send_mail_receipt(
+        self, pos_reference, email, body_from_ui, force=True
+    ):
+        order = self.search([("pos_reference", "=", pos_reference)])
+        if len(order) < 1:
+            return _("Error: no order found")
+        if order.email_receipt_sent:
+            return _("E-mail already sent")
```

Comment:
the returned values are not used. I think that a logger should be better. Don't you think ?